fuzz/fsfuzzer.py.data/fsfuzzer.yaml

```diff
@@ -0,0 +1,4 @@
+setup:
+	filesystem : !mux
+		default : 
+			fstype : iso9660
```

Comment:
please use soft-spaces instead of tabs. Also for this test it'd make sense to include all fstypes. For 0.7 it means:

```
$ ./fsfuzz --help
./fsfuzz cramfs|ext2|ext3|ext4|swap|iso9660|msdos|vfat|xfs
```